master/reference/kube-controllers/configuration.md

```diff
@@ -1,17 +1,24 @@
 ---
-title: Configuring the Calico policy controller
+title: Configuring the Calico Kubernetes Controllers
 ---
 
-The policy controller is primarily configured through environment variables. When running
-the policy controller as a Kubernetes pod, this is accomplished through the pod manifest `env`
+The kubernetes controllers are primarily configured through environment variables. When running
+the controllers as a Kubernetes pod, this is accomplished through the pod manifest `env`
```

Comment:
This makes it sound like multiple controllers are run in a single pod.....I think it would be better "a Kubernetes pod" --> "Kubernetes pods"

Reply:
Multiple controllers are run in a single pod, so that's correct.